### PR TITLE
Add inline documentation for various files.

### PR DESCRIPTION
**Pull request by GarryBullock**

### CODE REVIEW

**Comment by GarryBullock** [comment]
I just want to point out, circleci is running tests on PR's, and I dont know how haha.

File: web/test/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from bfex.common.utils import *
+
+
+class TestFacultyNames(object):
+    def test_validate_name(self):
+        valid_name = "JNelson.Amaral"
```
Comment (MandyMeindersma) [inline web/test/test_utils.py]: do we have to test when the name is ""?

**Comment by MandyMeindersma** [comment]
I just have that one random comment about the test, Other wise looks good :+1:

File: web/test/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        assert not FacultyNames.validate_name(invalid_name_2)
+
+    def test_build_directory_url(self):
+        valid_name = "Nelson.Amaral"
+        valid_name_2 = "JNelson.Amaral"
+
+        url_name = FacultyNames.build_url_name(valid_name)
+        assert url_name.islower()
+        assert "-" in url_name
+        assert "nelson-amaral"
```
Comment (morlick) [inline web/test/test_utils.py]: assert "nelson-amaral" doesn't really check anything, does it?
something to check instead could be assert "nelson-amaral" == url_name ?

**Pull request merged**